bot.py

```diff
@@ -455,6 +457,31 @@ async def handle_troop_search(self, message, search_term, lang):
 
         await message.channel.send(embed=e)
 
+    async def handle_talent_search(self, message, search_term, lang):
+        result = self.expander.search_talent_tree(search_term, lang)
+        if not result:
+            color = discord.Color.from_rgb(0, 0, 0)
+            e = discord.Embed(title='Talent search', color=color)
+            e.add_field(name=search_term, value='did not yield any result')
```

Comment:
should be a translated text

Reply:
yes that's true for all of those methods. I agree and will still accept the pull request

Reply:
obviously the current translation comes from game files, translating original bot text requires translators who speak each of the languages. It's not like I could add translations even if I wanted to.

Ideally translated texts should sit at different files (much like game files) and the code should only reference it , with fallback to default in case there's no translation yet. that way coders and translators can work independent of each other.